### PR TITLE
GOA-2328 display symbol in annotation response model

### PR DESCRIPTION
**Pull request by eddturner**

### CODE REVIEW

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/controller/AnnotationControllerIT.java [hunk 1]
```diff
@@ -66,12 +66,12 @@
...
     //Test Data
-    private static final String NOTEXISTS_ASSIGNED_BY = "ZZZZZ";
+    private static final String MISSING_ASSIGNED_BY = "ZZZZZ";
     private static final String RESOURCE_URL = "/QuickGO/services/annotation";
-    private static final String NOTEXISTS_GO_ID = "GO:0009871";
+    private static final String MISSING_GO_ID = "GO:0009871";
     private static final String INVALID_GO_ID = "GO:1";
     private static final String ECO_ID2 = "ECO:0000323";
-    private static final String NOTEXISTS_ECO_ID3 = "ECO:0000888";
+    private static final String MISSING_ECO_ID3 = "ECO:0000888";
```
Comment (rantunesboreas) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/controller/AnnotationControllerIT.java]: Why is the attribute called MISSING_ECO_ID3 and not  MISSING_ECO_ID?

File: annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImpl.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
         annotation.evidenceCode = annotationDocument.evidenceCode;
         annotation.reference = annotationDocument.reference;
         annotation.taxonId = annotationDocument.taxonId;
-
+        annotation.symbol = annotationDocument.dbObjectSymbol;
```
Comment (rantunesboreas) [inline annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImpl.java]: Could we rename dbObjectSymbol, to just symbol?

AnnotationFields.DB_OBJECT_SYMBOL should also  be renamed to SYMBOL

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImplTest.java [hunk 3]
```diff
@@ -32,6 +32,7 @@
     private static final List<String> WITH_FROM = Arrays.asList("GO:0036376", "GO:1990573");
     private static final String ASSIGNED_BY = "InterPro";
     private static final List<String> EXTENSIONS = Arrays.asList("occurs_in(CL:1000428)", "occurs_in(CL:1000429)");
+    private static final String DB_OBJECT_SYMBOL = "moeA5";
```
Comment (rantunesboreas) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImplTest.java]: Rename to SYMBOL

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/controller/AnnotationControllerIT.java (hunk 1 above)
Comment (eddturner) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/controller/AnnotationControllerIT.java]: done

File: annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImpl.java (hunk 2 above)
Comment (eddturner) [inline annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImpl.java]: done.

I also renamed annotation core's schema.xml field from `dbObjectSymbol` to `symbol`.

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImplTest.java (hunk 3 above)
Comment (eddturner) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/service/converter/AnnotationDocConverterImplTest.java]: done

**Pull request merged**